tests/integration/strange_data/test_nested_dataset.py

```diff
@@ -1,119 +0,0 @@
-from copy import deepcopy
```

Comment:
Question: is this one deprecated because the .json -> DataFrame burden is on MindsDB? Or is it something else?

Reply:
Yes, exactly.